src/www/ui/api/Models/Agent.php

```diff
@@ -122,16 +122,28 @@ public function getJSON($version=ApiVersion::V1)
    * @param integer $version
    * @return array
    */
-  public function getArray($version=ApiVersion::V1)
+  public function getArray($version=ApiVersion::V2)
   {
-    return [
-      'successfulAgents' => $this->getSuccessfulAgents(),
-      'uploadId' => $this->getUploadId(),
-      'agentName' => $this->getAgentName(),
-      'currentAgentId' => $this->getCurrentAgentId(),
-      'currentAgentRev' => $this->getCurrentAgentRev(),
-      'isAgentRunning' => $this->getIsAgentRunning()
-    ];
+    if($version==ApiVersion::V1){
+      return [
+        'successfulagents' => $this->getSuccessfulAgents(),
+        'uploadId' => $this->getUploadId(),
+        'agentname' => $this->getAgentName(),
+        'currentagentId' => $this->getCurrentAgentId(),
+        'currentagentRev' => $this->getCurrentAgentRev(),
+        'isagentrunning' => $this->getIsAgentRunning()
+      ];
+    }else{
+      return [
+        'successfulAgents' => $this->getSuccessfulAgents(),
+        'uploadId' => $this->getUploadId(),
+        'agentName' => $this->getAgentName(),
+        'currentAgentId' => $this->getCurrentAgentId(),
+        'currentAgentRev' => $this->getCurrentAgentRev(),
+        'isAgentRunning' => $this->getIsAgentRunning()
+      ];
+    }
+
```

Comment:
The V1 `return` keys do not exist, and neither are required (`successfulagents` and so on). Only the version needs to be passed in `getArray()`, and the model return syntax remains unchanged. Please revert.

Reply:
resolved